### DISCUSSION TITLE
How do I close the drawer during initialization

### DISCUSSION THREAD

**Discussion by dajuanmao0224** [quasarframework/quasar#11840] [General - Components / Directives / etc] [answered]
Dear author:
I am a beginner and am using  framework, the version is Quasar Framework v2.3.4.

I tried to build a drawer，Very smooth.
But when I want it to be closed by default in the initial state, it will not take effect anyway.
confirm that I will "const drawerright = Vue.ref(flase)" is set to false, it will still be changed to true.

Finally, I changed "v-model="drawerRight"" to "v-if="drawerRight"" Solved the problem。But the animation failed。

How do I initialize and close it？

**Comment by MilosPaunovic** [comment]
Hello, take a look at the documentation example:
[https://codepen.io/milospaunovic/pen/xxXYWrL?editors=101](https://codepen.io/milospaunovic/pen/xxXYWrL?editors=101)

If you still have problems, create a Codepen reproduction so someone could play with it.

**Reply by dajuanmao0224** [reply]
Thank you very much 
But this does not seem to be my question.
I hope not to show drawers at first, but the example still shows.
![image](https://user-images.githubusercontent.com/71390495/148362333-ff724ed5-d5f8-4f48-9360-d23de8c7a4e6.png)

**Comment by MilosPaunovic** [comment, marked as answer]
![Screenshot 2022-01-06 at 10 41 47](https://user-images.githubusercontent.com/62475782/148362722-ae545be2-74a5-408f-8448-644d94a48bbb.png)

[https://quasar.dev/layout/drawer#qdrawer-api](https://quasar.dev/layout/drawer#qdrawer-api)

You need to remove `show-if-above` and `breakpoint` props, or to control 'em the way you like:
[https://codepen.io/milospaunovic/pen/xxXYWrL?editors=101](https://codepen.io/milospaunovic/pen/xxXYWrL?editors=101)

**Reply by dajuanmao0224** [reply]
Thank you for your reply
It's very powerful
